auslib/db.py

```diff
@@ -1200,7 +1358,11 @@ def __init__(self, db, metadata, dialect, baseName):
         # update race, so they do not need to be versioned.
         super(SignoffsTable, self).__init__(db, dialect, versioned=False)
 
-    def insert(self, changed_by=None, transaction=None, dryrun=False, **columns):
+    def getPotentialRequiredSignoffs(self, affected_rows, transaction=None):
+        # Signoffs tables do not require signoffs
```

Comment:
Probably not. I mainly did this to match the base class signature, but that may be going away anyways per above. This class will probably end up unchanged if the required signoff verification moves to subclasses.